# Ground truth for ArUco tags in the real world
# Don't fool yourself, we will not publish GT here :)

tag1: [0,0,0]
tag2: [0,0,0]
tag3: [0,0,0]
tag4: [0,0,0]
tag5: [0,0,0]
tag6: [0,0,0]
tag7: [0,0,0]
tag8: [0,0,0]
tag9: [0,0,0]
tag10: [0,0,0]
tag11: [0,0,0]
tag12: [0,0,0]
tag13: [0,0,0]
tag14: [0,0,0]
